### ISSUE TITLE
/usr/local/lib/python3.6/dist-packages/sklearn/ensemble/_gb.py:1342: FutureWarning: The parameter 'presort' is deprecated and has no effect. It will be removed in v0.24. You can suppress this warning by not passing any value to the 'presort' parameter. We also recommend using HistGradientBoosting models instead.   FutureWarning)

### ISSUE THREAD

**Issue by wilions** [hyperopt/hyperopt-sklearn#160]
hi, I'm running a hyper-parameter optimization using hyperopt-sklearn on some scientific data on Colab. i've seen the following errors.

the code are as follows,

estim = HyperoptEstimator(regressor=any_regressor('rgr'),
                          preprocessing=any_preprocessing('pre'),
                          algo=tpe.suggest,
                          max_evals=100,
                          trial_timeout=100)

# Search the hyperparameter space based on the data

estim.fit(X_train, y_train)

# Show the results

print(estim.score(X_test, y_test))

print(estim.best_model())

the errors are

[01:56:24] WARNING: /workspace/src/objective/regression_obj.cu:152: reg:linear is now deprecated in favor of reg:squarederror.

  0%|          | 0/1 [00:00<?, ?it/s, best loss: ?]/usr/local/lib/python3.6/dist-packages/sklearn/ensemble/_gb.py:1342: FutureWarning: The parameter 'presort' is deprecated and has no effect. It will be removed in v0.24. You can suppress this warning by not passing any value to the 'presort' parameter. We also recommend using HistGradientBoosting models instead.
  FutureWarning)

**Comment by linminhtoo** [comment]
Yes, presort has been deprecated since v0.22 of sklearn. I just made a pull request to fix this.
